data_structures/arrays/permutations.py

```diff
@@ -45,6 +44,6 @@ def backtrack(start):
     import doctest
 
     # use res to print the data in permute2 function
```

Comment:
```suggestion
```
This comment isn't really necessary so we might as well get rid of it while we're happening to be fixing it up